### PR TITLE
fix: SCE-20 Navbar use font Inter

### PR DESCRIPTION
**Pull request by tihuan**
https://happy-duck.dev-sc.dev.czi.team/d/super-cool-spatial.cxg/

<details open>
  <summary>Argus Stack Details</summary>
  <br />
  <table>
    <tr>
      <th>Name</th>
      <td>thuang-fix-navbar-font-to-inter</td>
    </tr>
    <tr>
      <th>Short Name</th>
      <td>happy-duck</td>
    </tr>
    <tr>
      <th>Base URL</th>
      <td><a href="https://happy-duck.dev-sc.dev.czi.team" target="_blank">https://happy-duck.dev-sc.dev.czi.team</a></td>
    </tr>
  </table>
</details>

---

Screenshot 2024-09-20 at 10 19 35 AM

UPDATE (9/23): Restyled BetaChip to match Data Portal too!

Screenshot 2024-09-23 at 1 24 10 PM

### CODE REVIEW

**Review by kaloster — APPROVED** [review]

File: client/src/components/NavBar/style.ts [hunk 1]
```diff
@@ -20,6 +20,23 @@ export const Wrapper = styled.div`
   top: 0;
   width: 100%;
   z-index: 1;
+
+  /* Increase specificity for targeted elements specified in index.html and index_template.html */
```
Comment (kaloster) [inline client/src/components/NavBar/style.ts:24]: Possible to do something like instead?

```
  .bp5-button-text {
    font-family: "Inter", sans-serif;
  }
```

Comment (tihuan) [reply client/src/components/NavBar/style.ts:24]: Thanks for the suggestion, Ronen! 

I thought about having a smaller scope as well, but was worried that we might miss elements in the Navbar that also need Inter either now or later, thus the rationale of completing overriding the same scope of elements targeted in the `index.html` file

What do you think?

Thank you 😄

Comment (kaloster) [reply client/src/components/NavBar/style.ts:24]: Yes that makes sense to me! Thank you for clarifying 🙏🏽

Comment (tihuan) [reply client/src/components/NavBar/style.ts:24]: Awesome thanks for the great discussion 😄 💪 💪

**Pull request merged**